sympy/functions/elementary/tests/test_hyperbolic.py

```diff
@@ -721,6 +727,9 @@ def test_asech_series():
 def test_asech_rewrite():
     x = Symbol('x')
     assert asech(x).rewrite(log) == log(1/x + sqrt(1/x - 1) * sqrt(1/x + 1))
+    assert asech(x).rewrite(acosh) == acosh(1/x)
+    assert asech(x).rewrite(asinh) == Abs(asinh(sqrt(1/x**2 - 1)))
+    assert asech(x).rewrite(atanh) == Abs(atanh(x*sqrt(1/x**2 - 1)))
```

Comment:
Issues with branch, again, when `x > 1` -- do these need to be Piecewise?

Reply:
Maybe some of these rewrites that aren't needed for lambdify should be removed so that the main issue can be fixed.